action.yml

```diff
@@ -3,33 +3,51 @@ description: 'Documentation render tool for typo3 documentation'
 
 inputs:
   input:
-    description: 'Input directory'
+    description: 'Input directory (i.e. "Documentation")'
     required: false
     default: null
 
   working-directory:
-    description: 'Working directory'
+    description: 'Working directory (i.e. "./", or "t3docsproject")'
     required: false
     default: './'
 
   config:
-    description: 'Configuration file location'
+    description: 'Configuration file location (i.e. "Documentation/")'
     default: 'Documentation'
 
   output:
-    description: 'Output directory'
+    description: 'Output directory (i.e. "RenderedDocumentation/Result/project/0.0.0")'
```

Comment:
Should we really have this as example in the description? This nested structure is not necessary anymore, isn't it?

Reply:
I think that is the structure that is expected for the rest of the deployment process

Reply:
Ah, so once introduced, that's the way ;-) But the default value below is another folder name.

Reply:
I actually used this default value due to the action in `gh-render-action` https://github.com/TYPO3-Documentation/gh-render-action/blob/main/action.yml#L62: 

```
    - uses: TYPO3-Documentation/render-guides@main
      id: render-guides
      if: steps.enable_guides.outputs.GUIDES == 'true'
      with:
        working-directory: t3docsproject
        config: ./t3docsproject/Documentation/
        output: ./RenderedDocumentation/Result/project/0.0.0
```

So I wanted to make the example relate to how we actually use it.

Reply:
(That path is only used during the CI process, that's also why the version number and so forth is fixed)

Reply:
Ack.